src/ParserHelpers.php

```diff
@@ -40,10 +40,7 @@ function getFunctionLikeDeclarationFromParameter(Node\Parameter $node)
 
 function isFunctionLike(Node $node)
```

Comment:
we can get rid of this function then

Reply:
Okay, I though we're leaving it as a shortcut. Will do this tomorrow.